### PR TITLE
v3.1/reference/tools/download: add br download introduction

### PR DESCRIPTION
**Pull request by 3pointer**
### What is changed, added or deleted? 
add br download introduction after v3.1.0beta published

### Which version does your change affect? 
v.3.1

### CODE REVIEW

File: v3.1/reference/tools/download.md [hunk 1]
```diff
@@ -33,6 +33,8 @@ category: reference
 > **注意：**
 >
 > 下载链接中的 `{version}` 为 TiDB Lightning 的版本号。例如，`v3.0.5` 版本的下载链接为 `http://download.pingcap.org/tidb-toolkit-v3.0.5-linux-amd64.tar.gz`。也可以使用 `latest` 替代 `{version}` 来下载最新的未发布版本。
+>
+> 下载包在 v3.1.0-beta 以及之后的版本里包含 BR。
```
Comment (GregoryIan) [inline v3.1/reference/tools/download.md]:
```suggestion
```

File: v3.1/reference/tools/download.md [hunk 2]
```diff
@@ -33,6 +33,8 @@ category: reference
 > **注意：**
 >
 > 下载链接中的 `{version}` 为 TiDB Lightning 的版本号。例如，`v3.0.5` 版本的下载链接为 `http://download.pingcap.org/tidb-toolkit-v3.0.5-linux-amd64.tar.gz`。也可以使用 `latest` 替代 `{version}` 来下载最新的未发布版本。
+>
```
Comment (GregoryIan) [inline v3.1/reference/tools/download.md]:
```suggestion
```

File: v3.1/reference/tools/download.md [hunk 3]
```diff
@@ -22,9 +22,9 @@ category: reference
 >
 > 下载链接中的 `{version}` 为 TiDB 的版本号。例如，`v3.0.5` 版本的下载链接为 `http://download.pingcap.org/tidb-v3.0.5-linux-amd64.tar.gz`。也可以使用 `latest` 替代 `{version}` 来下载最新的未发布版本。
 
-## TiDB Lightning
+## TiDB Lightning 和 BR
```
Comment (GregoryIan) [inline v3.1/reference/tools/download.md]:
```suggestion
##  快速备份和恢复（BR）和 TiDB Lightning
```

File: v3.1/reference/tools/download.md [hunk 4]
```diff
@@ -22,9 +22,9 @@ category: reference
...
 
-使用下表中的链接下载 [TiDB Lightning](/v3.1/reference/tools/tidb-lightning/overview.md)：
+使用下表中的链接下载 [TiDB Lightning](/v3.1/reference/tools/tidb-lightning/overview.md) 和 [BR](/v3.1/how-to/maintain/backup-and-restore/br.md)：
```
Comment (GregoryIan) [inline v3.1/reference/tools/download.md]:
```suggestion
使用下表中的链接下载 [TiDB Lightning](/v3.1/reference/tools/tidb-lightning/overview.md) 和 [快速备份和恢复（BR）](/v3.1/how-to/maintain/backup-and-restore/br.md)：
```

**Comment by GregoryIan** [comment]
Rest LGTM

File: v3.1/reference/tools/download.md [hunk 5]
```diff
@@ -22,9 +22,9 @@ category: reference
 >
 > 下载链接中的 `{version}` 为 TiDB 的版本号。例如，`v3.0.5` 版本的下载链接为 `http://download.pingcap.org/tidb-v3.0.5-linux-amd64.tar.gz`。也可以使用 `latest` 替代 `{version}` 来下载最新的未发布版本。
 
-## TiDB Lightning
+##  快速备份和恢复（BR）和 TiDB Lightning
```
Comment (GregoryIan) [inline v3.1/reference/tools/download.md]:
```suggestion
##  快速备份和恢复 BR 和 TiDB Lightning
```

**Comment by lilin90** [comment]
@anotherrachel FYI. The changes in this PR apply to both `dev` and `v3.1`. @3pointer will update docs-cn `dev` later in another PR.

**Pull request merged**